### PR TITLE
Add twiml virtualagent examples.

### PR DESCRIPTION
**Pull request by sarahcstringer**
These were generated automatically using the twiml-generator tool. https://github.com/TwilioDevEd/twiml-generator

### CODE REVIEW

File: twiml/voice/connect/virtualagent/virtualagent.3.x.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const VoiceResponse = require('twilio').twiml.VoiceResponse;
+
+const response = new VoiceResponse();
+const connect = response.connect({
+    action: 'https://myactionurl.com/twiml'
+});
+connect.virtualAgent({
+    connectorName: 'project',
+    statuscallback: 'https://mycallbackurl.com'
```
Comment (craigsdennis) [inline twiml/voice/connect/virtualagent/virtualagent.3.x.js:9]: That casing right?

File: twiml/voice/connect/virtualagent/virtualagent.7.x.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import com.twilio.twiml.voice.Connect;
+import com.twilio.twiml.VoiceResponse;
+import com.twilio.twiml.voice.VirtualAgent;
+import com.twilio.twiml.TwiMLException;
+
+
+public class Example {
+    public static void main(String[] args) {
+        VirtualAgent virtualagent = new VirtualAgent.Builder().connectorName("project").statuscallback("https://mycallbackurl.com").build();
```
Comment (craigsdennis) [inline twiml/voice/connect/virtualagent/virtualagent.7.x.java:9]: Weird casing too....hmmmm

The ClassName (in PascalCase) should create an instanceObject (in camelCase)

Comment (craigsdennis) [reply twiml/voice/connect/virtualagent/virtualagent.7.x.java:9]: I'm not going to stop the approval, but maybe add an issue to TwiML Generator?

**Review by craigsdennis — APPROVED** [review]
LGTM

File: twiml/voice/connect/virtualagent/virtualagent.7.x.java (hunk 2 above)
Comment (sarahcstringer) [reply twiml/voice/connect/virtualagent/virtualagent.7.x.java:9]: That's true -- I should actually go test these first 🤦‍♀️ . I'll check out what's been generated.

File: twiml/voice/connect/virtualagent/virtualagent.3.x.js (hunk 1 above)
Comment (sarahcstringer) [reply twiml/voice/connect/virtualagent/virtualagent.3.x.js:9]: Tested this one and it's correct/works
```
❯ cat test.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;

const response = new VoiceResponse();
const connect = response.connect({
    action: 'https://myactionurl.com/twiml'
});
connect.virtualAgent({
    connectorName: 'project',
    statuscallback: 'https://mycallbackurl.com'
});

console.log(response.toString());

~
❯ node test.js
<?xml version="1.0" encoding="UTF-8"?><Response><Connect action="https://myactionurl.com/twiml"><VirtualAgent connectorName="project" statuscallback="https://mycallbackurl.com"/></Connect></Response>
```

Comment (sarahcstringer) [reply twiml/voice/connect/virtualagent/virtualagent.3.x.js:9]: oh nvm, javascript will allow anything. It should be camel case.

**Comment by sarahcstringer** [comment]
This was just me doing a typo in the original Twiml. Ugh.
But! It's fixed and I've tested that these work.

**Pull request merged**